platform/podjs/src/browserPod.ts

```diff
@@ -671,15 +673,14 @@ export class BrowserPod implements Pod {
 
     constructor() {
         window.addEventListener("load", async () => {
-            if (!window.manifestData) {
+            if (MANIFEST_DATA === {}) {
                 console.warn(
-                    `Unable to find feature manifest, navigation bar disabled.
-To get the navigation bar, expose the manifest's content as
-window.manifestData.`
+                    `Unable to find feature manifest, navigation bar disabled.`
```

Comment:
I guess that an equivalent change in the suggestion should be made, something like "add this and that to the rollup configuration", right?